website/blog/2022-12-28-databend-weekly-74.md

```diff
@@ -0,0 +1,179 @@
+
+**Query**
+
+- split fuse source to read data and deserialize ([#9353](https://github.com/datafuselabs/databend/pull/9353))
+- avoid io copy in read parquet data ([#9365](https://github.com/datafuselabs/databend/pull/9365))
+- add uncompressed buffer for parquet reader ([#9379](https://github.com/datafuselabs/databend/pull/9379))
+
+**Storage**
+
+- add read/write settings ([#9359](https://github.com/datafuselabs/databend/pull/9359))
+
+### Bug Fixes :wrench:
+
+**Format**
+
+- fix align_flush with header only ([#9327](https://github.com/datafuselabs/databend/pull/9327))
+
+**Settings**
+
+- use logical CPU number as default value of num_cpus ([#9396](https://github.com/datafuselabs/databend/pull/9396))
+
+**Processors**
+
+- the data type on both sides of the union does not match ([#9361](https://github.com/datafuselabs/databend/pull/9361))
+
+**HTTP Handler**
+
+- false alarm (warning log) about query not exists ([#9380](https://github.com/datafuselabs/databend/pull/9380))
+
+
+**Sqllogictest**
+
+- refactor sqllogictest http client and fix expression string like ([#9363](https://github.com/datafuselabs/databend/pull/9363))
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+#### Support Databend-Local
+
+Like [clickhouse-local](https://clickhouse.com/docs/en/operations/utilities/clickhouse-local/), databend-local can perform fast processing on local files without launching a databend cluster.
+
+```sql
+> export CONFIG_FILE=tests/local/config/databend-local.toml
+> cargo run --bin=databend-local -- --sql="SELECT * FROM tbl1" --table=tbl1=/path/to/databend/docs/public/data/books.parquet
+
+exec local query: SELECT * FROM tbl1
++------------------------------+---------------------+------+
+| title                        | author              | date |
++------------------------------+---------------------+------+
+| Transaction Processing       | Jim Gray            | 1992 |
+| Readings in Database Systems | Michael Stonebraker | 2004 |
+| Transaction Processing       | Jim Gray            | 1992 |
+| Readings in Database Systems | Michael Stonebraker | 2004 |
++------------------------------+---------------------+------+
+4 rows in set. Query took 0.015 seconds.
+```
+
+**Learn More**
+
+- [PR | support databend-local](https://github.com/datafuselabs/databend/pull/9282)
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+#### Compressing for Short Strings
+
+We have found that Databend may need to read more data than other databases such as Snowflake when dealing with some data items involving short strings.
+
+```sql
+SELECT SearchPhrase, MIN(URL), COUNT(*) AS c FROM hits WHERE URL LIKE '%google%' AND SearchPhrase <> '' GROUP BY SearchPhrase ORDER BY c DESC LIMIT 10;
+```
+
+Providing compression for short strings (URLs etc.) may help to improve the performance of such queries.
```

Comment:
Such queries might be more efficient if short strings (URLs, etc) are compressed.